src/frontend/src/common/api/index.tsx

```diff
@@ -69,33 +75,12 @@ async function apiResponse<T extends APIResponse>(
     }
     return [key, resultData];
   });
+
   return Object.fromEntries(convertedData);
 }
 
 /** Calls to specific API endpoints **/
 
-interface UserResponse extends APIResponse {
```

Comment:
Extracted auth related API calls to its own file `src/frontend/src/common/queries/auth.ts`